pkg/webui/lib/shared-messages.js

```diff
@@ -110,6 +116,7 @@ export default defineMessages({
   time: 'Time',
   traffic: 'Traffic',
   type: 'Type',
+  unknown: 'Unknown',
   updatedAt: 'Updated at',
```

Comment:
```suggestion
  updatedAt: 'Last Updated at',
```
I know this was not changed in this PR, but let's take the chance to make this message clearer.